src/modules/auth/LogIn.page.tsx

```diff
@@ -1,7 +1,22 @@
 import useLazyLoadResourceHook from '@shared/hooks/lazyLoadResource.hook';
 import { LogIn as LogInComponent } from './components/LogIn/LogIn.component';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { RootState } from '@store/index.store.ts';
 
 export default function LogIn(): JSX.Element {
+  const navigate = useNavigate();
+  const { token, isTrainer } = useSelector((state: RootState) => state.auth);
+  useEffect(() => {
+    if (!token) {
+      return;
+    }
+    if (isTrainer) {
+      navigate('/');
+    }
+    navigate('/');
+  }, [token, isTrainer, navigate]);
   useLazyLoadResourceHook({ folderName: 'auth', namespace: 'auth' });
```

Comment:
Can you set this as the first hook of this component? 
Since every page will have it... And it needs to load translation files, so, maybe it is best to do that first...

Reply:
Hm, do we need to load translation files if we navigate back to homescreen, we do not need translations?

Reply:
Everything needs translations, but the Home & Settings modules are here just to show the structure... We will delete them once we have, let's say one fully implemented module.

Also, this hook is not ideal... And I have some ideas on how we can make it better... But let's do that when we connect everything.